api/client/service/update_test.go

```diff
@@ -0,0 +1,23 @@
+package service
+
+import (
+	"testing"
+
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/engine-api/types/swarm"
+)
+
+func TestUpdateServiceCommandAndArgs(t *testing.T) {
+	flags := newUpdateCommand(nil).Flags()
+	flags.Set("command", "the new command")
+	flags.Set("args", "the new args")
```

Comment:
It might be good to throw a little escaping into the test case to make sure it's handled properly.